src/plugins/blocksmith/components/iframe-block-modal/index.jsx

```diff
@@ -68,7 +68,8 @@ export default class IframeBlockModal extends React.Component {
     this.state = {
       align: block.get('align'),
       data: block.get('data') || {},
-      hasManualDimensions: block.has('height') && block.has('width'),
+      hasManualDimensions: block.has('height') || block.has('width'),
+      hasScrollingEnabled: false,
```

Comment:
@nanetteCodes - please use `scrollingEnabled` for the literal here since we are not doing any computations, also change the value to `block.get('scrolling_enabled')`